src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  // Поділ  рядків крапками з комою
+  const declarations = styles.split(';');
```

Comment:
The variable name `declarations` is clear and descriptive, which is good. However, it's important to note that the `split` method will include empty strings in the array if there are consecutive semicolons. It would be nice to filter out these empty strings to avoid unnecessary iterations in the `forEach` loop.